sky/jobs/utils.py

```diff
@@ -550,18 +550,25 @@ def should_keep_logging(status: managed_job_state.ManagedJobStatus) -> bool:
             if managed_job_status.is_failed():
                 job_msg = ('\nFailure reason: '
                            f'{managed_job_state.get_failure_reason(job_id)}')
-            log_file = managed_job_state.get_local_log_file(job_id, None)
-            if log_file is not None:
-                with open(os.path.expanduser(log_file), 'r',
-                          encoding='utf-8') as f:
-                    # Stream the logs to the console without reading the whole
-                    # file into memory.
-                    start_streaming = False
-                    for line in f:
-                        if log_lib.LOG_FILE_START_STREAMING_AT in line:
-                            start_streaming = True
-                        if start_streaming:
-                            print(line, end='', flush=True)
+            log_file_exists = False
+            task_ids = managed_job_state.get_all_task_ids(job_id)
+            for job_task_id in task_ids:
```

Comment:
How do people distinguish multiple tasks under a same job? Also, how do people find the previous log when the job has not finished yet?

Reply:
To be improved:
1. Add a separation line, e.g. `== task 2 ==` between the logs for different tasks.
2. Support getting logs for a specific task. https://github.com/skypilot-org/skypilot/issues/5485